collector/node_conn_bloxroute.go

```diff
@@ -131,7 +131,7 @@ func (nc *BlxNodeConnection) connect() {
 
 		// fmt.Println("got message", string(nextNotification))
 		var txMsg common.BlxRawTxMsg
-		err = json.Unmarshal(nextNotification, &txMsg)
+		err = json.Unmarshal(nextNotification, &txMsg) //nolint:musttag
```

Comment:
Out of curiosity, what's going on here?

Reply:
nice catch, and good q! https://github.com/go-simpler/musttag

meaning the fields here should all be `json` tagged: https://github.com/flashbots/mempool-dumpster/blob/cc574e013e6cb28bb6a827e53a1296ab515031cc/common/types.go#L7-L14